pkg/framework/framework.go

```diff
@@ -41,6 +42,8 @@ var (
 	Timeout              = time.Second * 600
 	CleanupRetryInterval = time.Second * 1
 	CleanupTimeout       = time.Second * 5
+	RestConfig rest.Config
```

Comment:
@mkrutov I do not think that the RestConfig can be a public variable,
because the Framework allows you to specify multiple contexts (using --context).
If you do so, then you cannot use a single RestConfig variable. 

In case you need it, might be better to add it to the ClientSet type, so it can be tied
to the related ContextData type.